edx/edx-enterprise
# Manage Learners: missed migration and fixed test

**Description:** This template fixes missing migration in #8 

**JIRA:** [ENT-48](https://openedx.atlassian.net/browse/ENT-48)

**Merge deadline:** Monday 21st

**Testing instructions:**

There are no user facing changes - unique constraint was already enforced in "Add Learner" form validation.

So, running the migrations, smoke testing Django admin interface (create new EC, edit old one, etc.) and running the test suite should be enough.

**Reviewers:**
- [x] @asadiqbal08 and/or @saleem-latif 

**Merge checklist:**

**Post merge:**
      finished.

**Author concerns:**
